Examples/MAX32665/Dual_Core/project.mk

```diff
@@ -0,0 +1,20 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://analogdevicesinc.github.io/msdk/USERGUIDE/#build-system
+
+# **********************************************************
+
+# Add your config here!
+
+# Build with the necessary Core1 startup/system files.
+ARM_DUALCORE=1
+
+# Separate directories for Core 0 and Core 1 code.
+VPATH += Core0
```

Comment:
Might not be necessary to separate the directories since both cores share the same arch.  You could even define `main` and `main_core1` in the same file.

Reply:
Yeah, mainly did this so users can organize their code between the two cores.